### PR TITLE
Keep Mycroft logs from VK test runs

### PR DESCRIPTION
**Pull request by krisgesling**
This extracts the standard files from /var/log/mycroft from the Docker container after the Voight Kampff test runs to provide extra debugging information. Particularly useful if the issue being reported cannot be replicated locally as we've seen with the stop method on the News and Singing Skills.

Currently the links for the log files are provided only in the failure email. Presumably there's no need to see the logs if everything passed.

### To test
See the [Jenkins console for PR-1360](https://hudson.mycroft.team/job/mycroft-skills/view/change-requests/job/PR-1360/22/console) that got replayed with this configuration, and check the related email from that job.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,189 +1,213 @@
...
-        disableConcurrentBuilds()
-        // Only keep the last 5 builds.
-        buildDiscarder(logRotator(numToKeepStr: '5'))
-    }
-    stages {
-        stage('Run Integration Tests') {
-            when {
-                anyOf {
-                    changeRequest()
+    pipeline {
```
Comment (forslund) [inline Jenkinsfile]: Is the indentation change intended?

**Review by forslund — COMMENTED** [review]
Looks good, the complete indentation change makes it hard to review since all lines are reported as changed.

But the log volume  Docker arguments looks correct and the logfiles are removed after each job...
Perhaps add the link to the logs to the VK test PR comment?

**Comment by forslund** [comment]
Also seems like an old commit tagged along on this PR...

File: Jenkinsfile (hunk 1 above)
Comment (krisgesling) [reply Jenkinsfile]: Nope, fixed now.

**Comment by krisgesling** [comment]
All fixed now. 
- rebased off 20.08 branch
- fixed the indentation
- added log links to PR comment

I've also triggered one more replay of PR-1360 using this Jenkinsfile as a final test:
https://hudson.mycroft.team/job/mycroft-skills/view/change-requests/job/PR-1360/27/

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/skills/PR-1391))

**Comment by forslund** [comment]
Looks good! Did you find out why Jenkins reported unstable?

**Comment by krisgesling** [comment]
Somehow it had a single test from the Web-Launcher Skill that failed. Not sure why:
1. the test was included at all
2. it reported unstable instead of fail

That Skill isn't mentioned at all in the [console output](https://hudson.mycroft.team/job/mycroft-skills/job/PR-1391/1/console), so I'm wondering if the docker image didn't get fully cleaned from the last run or something.

**Review by chrisveilleux — APPROVED** [review]

**Pull request merged**